python/setup.py

```diff
@@ -27,9 +27,9 @@
     "python-dateutil>=2.1",
     "six>=1.10",
     "urllib3>=1.23",
-    "Authlib<=0.16.0",
```

Comment:
Is it really only this specific version?

Reply:
I just mean if sth along
```
Authlib>=1.0.1,<=2.0.0
```
wouldn't be better?

Reply:
Oh good point. Let me change that.